src/components/dialogs/move-dialog.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { elementType } from '@gridsuite/commons-ui';
+import DirectorySelector from './directory-selector';
+import { useIntl } from 'react-intl';
+
+const MoveDialog = ({ open, onClose, items }) => {
+    const intl = useIntl();
+
+    //TODO : change nbElement
+    return (
+        <>
+            <DirectorySelector
+                types={elementType.DIRECTORY}
+                open={open}
+                onClose={onClose}
+                title={intl.formatMessage({ id: 'moveItemTitle' })}
```

Comment:
can also be done with <FormattedMessage id={...} values={{...}}/>
(just a remark)

Reply:
prefered using intl hooks since it's used several time in the component